### DISCUSSION TITLE
Translation Admin section in crowdin

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#235] [Q&A] [answered]
Hi, I've create a crowdin account to support translation. 
But do not find e.g. the context of the admin section ?

What do I wrong ?

**Comment by mgineer85** [comment, marked as answer]
Not very sure what you mean. Usually after you signed up, you click the links in the contribution information and this should bring you to crowdin look similar like this: 

![image](https://github.com/photobooth-app/photobooth-app/assets/3868450/bd4f9c27-bf78-433d-86f5-c3c2250c7132)

There you select the language and choose to translate (which is done automatically) or choose proofread to verify/modify automated translation.

Please not that the configuration and some minor strings are not covered by translation currently.

**Reply by Striker62** [reply]
ok that explain why i can not see the content from the config page
